api/core/workflow/nodes/http_request/http_request_node.py

```diff
@@ -11,19 +12,16 @@
 from core.workflow.entities.variable_pool import VariablePool
 from core.workflow.nodes.base_node import BaseNode
 from core.workflow.nodes.http_request.entities import (
-    MAX_CONNECT_TIMEOUT,
-    MAX_READ_TIMEOUT,
-    MAX_WRITE_TIMEOUT,
     HttpRequestNodeData,
     HttpRequestNodeTimeout,
 )
 from core.workflow.nodes.http_request.http_executor import HttpExecutor, HttpExecutorResponse
 from models.workflow import WorkflowNodeExecutionStatus
 
 HTTP_REQUEST_DEFAULT_TIMEOUT = HttpRequestNodeTimeout(
-    connect=min(10, MAX_CONNECT_TIMEOUT),
-    read=min(60, MAX_READ_TIMEOUT),
-    write=min(20, MAX_WRITE_TIMEOUT),
+    connect=min(10, dify_config.HTTP_REQUEST_MAX_CONNECT_TIMEOUT),
+    read=min(60, dify_config.HTTP_REQUEST_MAX_READ_TIMEOUT),
+    write=min(20, dify_config.HTTP_REQUEST_MAX_WRITE_TIMEOUT),
```

Comment:
I'm wondering why we use the minimum value for timeouts here? 
Minimum value will never make the  configs applicable. 
Shall we change to use maximum value to allow it configurable and applicable?

cc previous related author @laipz8200

Reply:
I think using `max` here is ok.

Reply:
Shall we raise a separate PR to fix it? Let's merge this PR first.

Reply:
I think it’s better to use `max` here, but this change shouldn’t be made in this PR.

I'll merge this PR first.